### PR TITLE
refactor(ui): rename element to zinc token

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-774

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [\[PR\] staging - refactor(ui): rename element to zinc token - 2023-08-22T13:45:11Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/3f6eb108-1f41-413f-b103-aed94ebbf1c1/services/general)

 Another comment will be posted when deployments are terminated

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://773-console-staging.zc531a994.rustrocks.cloud)

File: libs/pages/logs/environment/src/lib/ui/sidebar-status/sidebar-status.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ export function SidebarStatus(props: SidebarStatusProps) {
           4{' '}
           <Tooltip side="right" content="Number of services deployed in parallel on each pipeline stage">
             <span className="flex items-center">
-              <Icon className="cursor-pointer ml-1 text-xs text-element-light-zinc-300" name="icon-solid-circle-info" />
+              <Icon className="cursor-pointer ml-1 text-xs text-text-zinc-300" name="icon-solid-circle-info" />
```
Comment (ctjhoa) [inline libs/pages/logs/environment/src/lib/ui/sidebar-status/sidebar-status.tsx:43]: There is something wrong in this case. You should search for similar cases

**Review by ctjhoa — CHANGES_REQUESTED** [review]

File: libs/pages/logs/environment/src/lib/ui/sidebar-status/sidebar-status.tsx (hunk 1 above)
Comment (RemiBonnet) [reply libs/pages/logs/environment/src/lib/ui/sidebar-status/sidebar-status.tsx:43]: nice catch, just one case like this, fixed !

**Review by ctjhoa — APPROVED** [review]

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**